File: Assets/ShootingRPGKit/CsharpExample/ScriptCS/OtherScript/PushbackCollider.cs
using UnityEngine;
using System.Collections;

public class PushbackCollider : MonoBehaviour {
	public float backSpeed = 4.0f;
	public string pushTag = "Player";

	void OnTriggerStay(Collider other){
		if(other.gameObject.tag == pushTag){
			other.GetComponent<CharacterController>().Move(transform.rotation * Vector3.back * backSpeed *Time.deltaTime);
		}
	}

}

File: Assets/ShootingRPGKit/Scripts/OtherScript/PushbackCollider.js
#pragma strict
var backSpeed : float = 4.0;
var pushTag : String = "Player";

function OnTriggerStay (other : Collider) {
	if(other.gameObject.tag == pushTag){
		other.GetComponent(CharacterController).Move(transform.rotation * Vector3.back * backSpeed *Time.deltaTime);
	}

}